omnibus/config/templates/datadog-agent/upstart_redhat.conf.erb

```diff
@@ -1,6 +1,6 @@
 description "Datadog Agent"
 
-start on started networking
+start on started networking or started network
```

Comment:
This script is only used on RHEL6/CentOS6/other distros based on RHEL6. So I think it's safe to remove the `started networking` part entirely

Reply:
Good to know. I added both options to the other upstart templates as they don't appear to be RHEL specific